### ISSUE TITLE
Language Chooser template does not respect modes

### ISSUE THREAD

**Issue by leesolway** [divio/django-cms#4367]
{% language_chooser %} template tag (found in menu_tags.py) adds a context variable languages. 

```
context['languages'] = languages
```

The first thing language_choose.html template does is to override the language variable breaking the chooser modes functionality. When moving the following line the template renders as expected:

```
{% get_available_languages as languages %}
```

**Comment by leesolway** [comment]
Amended title

**Comment by yakky** [comment]
@leesolway this has been fixed in #4217. Fix is going to be released soon for both 3.0 and 3.1
